etl/src/main/java/org/pih/hivmigration/etl/sql/InfantMigrator.groovy

```diff
@@ -0,0 +1,142 @@
+    def void migrate() {
+
+        executeMysql("Create Infants Staging Table", '''
+            create table hivmigration_infants (
+                person_id int PRIMARY KEY AUTO_INCREMENT,
+                source_infant_id int,
+                mother_patient_id int,
+                person_uuid char(38),
+                infant_code varchar(20),
+                first_name varchar(100),
+                last_name varchar(100),
+                gender varchar(50),
+                birthdate date
+            );
+        ''')
+
+        setAutoIncrement("hivmigration_infants", "(select (max(person_id)+1) from person)")
+
+        loadFromOracleToMySql('''
+                insert into hivmigration_infants (
+                    source_infant_id,
+                    mother_patient_id,
+                    infant_code,
+                    first_name,
+                    last_name,
+                    gender,
+                    birthdate
+                ) values (?, ?, ?, ?, ?, ?, ?)
+            ''', '''
+                select 
+                    i.INFANT_ID,
+                    i.MOTHER_PATIENT_ID,
+                    i.INFANT_CODE, 
+                    i.FIRST_NAME,
+                    i.LAST_NAME, 
+                    upper(i.GENDER), 
+                    i.BIRTH_DATE  			
+                from HIV_INFANTS i
+        ''')
+
+        executeMysql("Add UUIDs", "update hivmigration_infants set person_uuid = uuid();")
+
+        executeMysql("Load to person table", '''
+            insert into person
+              (person_id, uuid, gender, birthdate, creator, date_created)
+            select
+              i.person_id,
+              i.person_uuid,
+              COALESCE(i.gender, 'U'),  # most of the data is missing gender -- default to U
+              i.birthdate,
+              1,
+              date_format(curdate(), '%Y-%m-%d %T')
+            from
+              hivmigration_infants i 
+            order by i.source_infant_id;
+        ''')
+
+        executeMysql("Load infant names to person_name table", '''
+            insert into person_name (
+                person_id, uuid, given_name, family_name, preferred, creator, date_created
+            )
+            select
+              i.person_id,
+              uuid(),
+              i.first_name,
+              i.last_name,
+              1,
+              1,
+              date_format(curdate(), '%Y-%m-%d %T')
+            from
+              hivmigration_infants i
+        ''')
+
+        executeMysql("Load infants into patient table", '''
+            insert into patient (patient_id, creator, date_created)
+            select
+              i.person_id,
+              1,
+              date_format(curdate(), '%Y-%m-%d %T')
+            from
+              hivmigration_infants i
+            order by i.source_infant_id;
+        ''')
+
+        // TODO: depends on zlemrid from Patient migration (https://pihemr.atlassian.net/browse/UHM-3145)
+        /*
```

Comment:
Cool, I can take care of this when I finish Patients (hopefully today)... can you assign this ticket to me once you merge to remind me to take care of this?

Reply:
Sure thing, thanks!